src/Facades/CurrencyHelper.php

```diff
@@ -6,10 +6,13 @@
 use Illuminate\Support\Facades\Facade;
 
 /**
+ * // phpcs:disable
```

Comment:
why do we need to disable `phpcs` here?

Reply:
Lines are too long and I personally don't like to put phpdoc for facade methods. But I can change them If you want :)

Reply:
I see, that's ok we can keep it